app/models/ext_management_system.rb

```diff
@@ -232,6 +232,8 @@ def validate_zone_not_maintenance_when_ems_enabled?
            :port=,
            :security_protocol,
            :security_protocol=,
+           :path,
+           :path=,
```

Comment:
`path` is already defined by memoist, we should prefix this with `endpoint`

You can do:
```
delegate :path, :path=, :to => :default_endpoint, :prefix => "endpoint", :allow_nil => true
```

Reply:
Done